src/main/java/org/broadinstitute/hellbender/utils/Utils.java

```diff
@@ -254,6 +254,39 @@ public static String join(final String separator, final double[] doubles) {
     }
 
     /**
+     * Splits a String using indexOf instead of regex to speed things up.
+     *
+     * @param str the string to split.
+     * @param delimiter the delimiter used to split the string.
+     * @return an array of tokens.
+     */
+    public static ArrayList<String> split(final String str, final String delimiter) {
```

Comment:
do not return `ArrayList`. Return `List`

Also, add argument checks for both arguments (i'm assuming they can't be null).
Add unit tests for this method